homeassistant/components/onewire/sensor.py

```diff
@@ -175,11 +180,22 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     # We have an owfs mounted
     else:
-        for family_file_path in glob(os.path.join(base_dir, "*", "family")):
+        families = glob(os.path.join(base_dir, "*", "family"))
+        while families:
+            family_file_path = families.pop()
             with open(family_file_path, "r") as family_file:
                 family = family_file.read()
             if "EF" in family:
                 continue
+            if "1F" in family:  # coupler
+                for branch in ["main", "aux"]:
+                    new_families = glob(
+                        os.path.join(
+                            os.path.split(family_file_path)[0], branch, "*", "family"
+                        )
+                    )
+                    families.extend(new_families)
```

Comment:
We don't want to extend the owfs part of this integration until the integration uses a 3rd party library for the device specific interface code and data parsing.